hello_vsi/FVP_Corstone_SSE-315.cfg

```diff
@@ -0,0 +1,7 @@
+# Parameters:
+# instance.parameter=value       #(type, mode) default = 'def value' : description : [min..max]
+#---------------------------------------------------------------------------------------------------
+mps4_board.visualisation.disable-visualisation=1      # (bool  , init-time) default = '0'          : Enable/disable visualisation
+vis_hdlcd.disable_visualisation=0                     # (bool  , init-time) default = '0'          : Enable/disable visualisation
+mps4_board.DISABLE_GATING=1                           # (bool  , init-time) default = '0'          : Disable Memory gating logic

```

Comment:
mps3_board.DISABLE_GATING=1  can be removed, as default setting shall work as well

Reply:
Removed